README.md

```diff
@@ -29,14 +29,18 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. Describe the biggest difference between `.forEach` & `.map`.
-
+    .forEach does not create a new array and .map does.
 2. What is the difference between a function and a method?
-
+    A method itself is a function but a function is only called a method when the function is a property of an object.
 3. What is closure?
-
+    A closure is a function that lives within a bubble and can keep a memory of itself.
```

Comment:
Here's my answer for the same question
`
Closure is a type of scope. If the compiler can't find a variable it's looking for on the local scope it "looks up" to the scope above it. The way JS passes variables down through scope is called closure.
`